mpact/tests/test_periodic_tasks.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.test import TestCase
+
+from mpact.models import GroupChat, ScheduledMessage
+from mpact.scheduling import get_periodic_tasks_for_group, rebuild_schedule_for_group
+
+
+class PeriodicTaskTestCase(TestCase):
+    GROUP_1_ID = 12345
+    GROUP_2_ID = 54321
```

Comment:
Any reason it shouldn't pull these from `test.util` also?

Reply:
Oh, yeah I originally thought `test.util` would be shared between the two tests but then it didn't end up being worth it. probably what makes more sense is to just pull the constants (maybe everything) out of there and pull it back in the spreadsheet test.